applications/GeoMechanicsApplication/tests/test_parameter_field.py

```diff
@@ -65,9 +66,27 @@ def test_parameter_field_with_python_umat_parameters(self):
 
         # copy user defined python script to installation folder
         new_custom_script_path = os.path.join(os.path.dirname(KratosGeo.__file__), "user_defined_scripts")
-        shutil.copy(custom_python_file,new_custom_script_path)
-
-        # run simulation
+        try:
+            shutil.copy(custom_python_file, new_custom_script_path)
+        except shutil.SameFileError as e:
+             print('Source and destination represents the same file. %s' % e)
```

Comment:
I would suggest to use Python f-strings, since it's syntax is more generally applicable (e.g. you don't need the format specifier `%s`) and therefore slightly easier to read:
```suggestion
             print(f'Source and destination represents the same file. {e}')
```

This suggestion also applies to the other `print` statements that you've added.

Reply:
done